fpga/src/main/resources/vcu118/sdboot/sd.c

```diff
@@ -8,10 +8,12 @@
 #define DEBUG
 #include "kprintf.h"
 
-#define MAX_CORES 8
-
-// A sector is 512 bytes, so ((1 << 11) * 512) = 1 MiB
-#define PAYLOAD_SIZE	(16 << 11)
+// Total payload in B
+#define PAYLOAD_SIZE_B (30 << 20) // default: 30MiB
```

Comment:
How much extra room is there here?

Reply:
The Linux binary with FireMarshal is around 20-25MiB so 5MiB room.